README.md

```diff
@@ -84,18 +84,65 @@ Please check [Xendit API Reference](https://xendit.github.io/apireference/).
 JDK 1.7 or later.
 
 ## Installation
-Maven:
+####Maven
+
+Add these lines of code in your `pom.xml`
 ```
 <dependency>
   <groupId>com.xendit</groupId>
   <artifactId>xendit-java-lib</artifactId>
   <version>SELECTED_VERSION</version>
-  <type>pom</type>
 </dependency>
-```
 
-Gradle:
-```
+<repositories>
+  <repository>
+    <snapshots>
+      <enabled>
+        false
+      </enabled>
+    </snapshots>
+    <id>
+      bintray-xendit-android
+    </id>
+    <name>
+      bintray
+    </name>
+    <url>
+      https://dl.bintray.com/xendit/android
+    </url>
+  </repository>
+</repositories>
+<pluginRepositories>
+  <pluginRepository>
+    <snapshots>
+      <enabled>
+        false
+      </enabled>
+    </snapshots>
+    <id>
+      bintray-xendit-android
+    </id>
+    <name>
+      bintray-plugins
+    </name>
+    <url>
+      https://dl.bintray.com/xendit/android
+    </url>
+  </pluginRepository>
+</pluginRepositories>
+```
+
+####Gradle
+
+Add these lines in your `build.gradle`
+```
+...
+repositories {
+	maven {
+		url  "https://dl.bintray.com/xendit/android"
```

Comment:
why is this android?

Reply:
It is a legacy repository. We will fix this after we publish the version 2.0 under the `xendit-java` name